Resources/Prototypes/Entities/items/clothing/IDs.yml

```diff
@@ -3,46 +3,74 @@
   id: IDCardStandard
   name: Identification Card
   description: A card necessary to access various areas aboard the station
+  abstract: true
   components:
   - type: Sprite
-    sprite: Clothing/id_cards.rsi
-    state: civilian
-  - type: Icon
-    sprite: Clothing/id_cards.rsi
-    state: civilian
+    drawdepth: FloorObjects
```

Comment:
`FloorObjects` is not the right layer to use at all. Honestly right now all the layers are a broken mess anyways so I wouldn't bother with trying to fix specifically ID cards.